blake2fmodexpdata/columns.lisp

```diff
@@ -1,18 +1,18 @@
 (module blake2fmodexpdata)
 
 (defcolumns 
-  (STAMP :byte)
-  (ID :i48)
-  (PHASE :byte)
-  (INDEX :byte :display :dec)
-  (INDEX_MAX :byte :display :dec)
-  (LIMB :i128 :display :bytes)
-  (IS_MODEXP_BASE :binary@prove)
-  (IS_MODEXP_EXPONENT :binary@prove)
-  (IS_MODEXP_MODULUS :binary@prove)
-  (IS_MODEXP_RESULT :binary@prove)
-  (IS_BLAKE_DATA :binary@prove)
-  (IS_BLAKE_PARAMS :binary@prove)
-  (IS_BLAKE_RESULT :binary@prove))
-
+  (STAMP                :byte)
+  (ID                   :i48)
```

Comment:
Hub_Stamp is the only stamp to be :i48 and not :i32 ?

Reply:
I think we can go either way for the ID. IIRC the STAMP counts without gaps. But even with current limitations (600 rounds in total for BLAKE if memory serves) that `:byte` is insufficient.